app/cli/src/main/scala/org/bitcoins/cli/ConsoleCli.scala

```diff
@@ -91,8 +91,18 @@ object ConsoleCli {
         ),
       cmd("getbalance")
         .hidden()
-        .action((_, conf) => conf.copy(command = GetBalance))
-        .text("Get the wallet balance"),
+        .action((_, conf) => conf.copy(command = GetBalance(false)))
+        .text("Get the wallet balance")
```

Comment:
nit: We should probably start adding config flags in the help message, which is what the text message is I'm guessing?

Reply:
I think that is out of the scope of this PR but I will make an issue for it.